src/cmd/rules.go

```diff
@@ -8,20 +8,21 @@ import (
 	"strings"
 
 	"github.com/VKCOM/noverify/src/cmd/embeddedrules"
-	"github.com/VKCOM/noverify/src/linter"
+	"github.com/VKCOM/noverify/src/ir"
 	"github.com/VKCOM/noverify/src/rules"
 )
 
-func InitEmbeddedRules(config *linter.Config, p *rules.Parser, filter func(r rules.Rule) bool) ([]*rules.Set, error) {
-	ruleSets, err := parseEmbeddedRules(p)
+func AddEmbeddedRules(rset *rules.Set, p *rules.Parser, filter func(r rules.Rule) bool) ([]*rules.Set, error) {
+	embeddedRules, err := parseEmbeddedRules(p)
 	if err != nil {
 		return nil, err
 	}
 
-	for _, rset := range ruleSets {
-		appendRuleSet(config, rset, filter)
+	for _, embeddedRule := range embeddedRules {
```

Comment:
It's not a rule, but a rule set.
Maybe remove the embeddedRule variable and use `embeddedRules[i]` to pass it as an argument?

Reply:
Can be renamed to `embeddedRuleSet` and `embeddedRules` to `embeddedRuleSets`.